### PR TITLE
Entities

### PR DESCRIPTION
**Pull request by MarkPikanovsky**

### CODE REVIEW

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/Material.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "material")
+public enum Material {
+    BOOK,
```
Comment (aleksashyn) [inline core/src/main/java/com/lms/system/teacher/crm/core/entity/Material.java]: Enum should be separate class like a Material Type

Comment (MarkPikanovsky) [reply core/src/main/java/com/lms/system/teacher/crm/core/entity/Material.java]: done

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/Lesson.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import javax.persistence.JoinTable;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Entity
+@Table(name = "lesson")
+@Data
+
```
Comment (aleksashyn) [inline core/src/main/java/com/lms/system/teacher/crm/core/entity/Lesson.java]: Why you put this empty line?

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/Lesson.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+public class Lesson {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "portfolio_id")
+    private Long portfolioId;
```
Comment (aleksashyn) [inline core/src/main/java/com/lms/system/teacher/crm/core/entity/Lesson.java]: Please use class LessonsPortfolio instead Long

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/LessonMaterial.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "material")
+public class LessonMaterial {
```
Comment (aleksashyn) [inline core/src/main/java/com/lms/system/teacher/crm/core/entity/LessonMaterial.java]: I think that Material is better

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/LessonMaterial.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "type", length = 10)
+    @Enumerated(EnumType.STRING)
+    private Material materialType;
```
Comment (aleksashyn) [inline core/src/main/java/com/lms/system/teacher/crm/core/entity/LessonMaterial.java]: I think that MaterialType is bettre

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/Lesson.java (hunk 2 above)
Comment (MarkPikanovsky) [reply core/src/main/java/com/lms/system/teacher/crm/core/entity/Lesson.java]: removed

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/Lesson.java (hunk 3 above)
Comment (MarkPikanovsky) [reply core/src/main/java/com/lms/system/teacher/crm/core/entity/Lesson.java]: done

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/LessonMaterial.java (hunk 4 above)
Comment (MarkPikanovsky) [reply core/src/main/java/com/lms/system/teacher/crm/core/entity/LessonMaterial.java]: ok,changed

File: core/src/main/java/com/lms/system/teacher/crm/core/entity/LessonMaterial.java (hunk 5 above)
Comment (MarkPikanovsky) [reply core/src/main/java/com/lms/system/teacher/crm/core/entity/LessonMaterial.java]: +

**Pull request merged**